content/en/platform/corda/4.10/enterprise/node/operating/monitoring-and-logging/opentelemetry.md

```diff
@@ -0,0 +1,103 @@
+3. Replace `PATH-TO-OT-JAVAAGENT` with the full path to where you placed the OpenTelemetry Java agent.
+4. Replace `YOUR-SERVICE-NAME` with the service name you would like the Corda node to be identified as.
+
+{{< note >}}
+This method of linking the SDK with the Corda node enables OpenTelemetry automated instrumentation, which means some other third-party libraries included with Corda will also generate spans. See the [OpenTelemetry](https://opentelemetry.io/docs/concepts/instrumenting-library/) website for details on which libraries are enabled.
+{{< /note >}}
+
+
+## Setting up OpenTelemetry
+
+Corda has been tested with the OpenTelemetry collector and a Jaeger backend. Refer to the [OpenTelemetry](https://opentelemetry.io/docs/collector/) and [Jaeger](https://www.jaegertracing.io/) websites for details on how to setup a collector and Jaeger backend.
+
+Corda OpenTelemetry can also be enabled or disabled via a node configuration parameter, such as `telementry.openTelemetryEnabled = true`.
+
+If this setting is true and the OpenTelemetry SDK has been linked to Corda as described above, the node will generate spans. If this setting is false, the node won't generate spans, even if the node is linked with the SDK.
+
+## Client API
+
+OpenTelemetry span generation is also incorporated into the RPC client API. If you want your Corda client code to generate spans, you need to include the OpenTelemetry SDK as a dependency. The methods described above only apply to the Corda node. 
+
+<!-- (TODO: Give example of dependencies needed). -->
+
+## Creating your own Spans
+
+The OpenTelemetry API may be used in your flows and in your client code to create spans and baggage. To get an instance of the OpenTelemetry API in a flow, make the following call:
+`val openTelemetry: OpenTelemetry? = serviceHub.telemetryService.getTelemetryHandle(OpenTelemetry::class.java)`.
+
+From the client API, where RPC is a CordaRPCConnection, you would use the following:
+`val openTelemetry: OpenTelemetry? = rpc.getTelemetryHandle(OpenTelemetry::class.java)`.
+
+When creating your own spans, you can also create your own baggage. If you create your own baggage, it will also be sent to other nodes, and you can specify if you want this baggage to be copied to span tags. If you do, all of the spans involved in the transaction for that node will also get a copy of the baggage. This can be enabled with the following parameter:
+`telementry.copyBaggageToTags = true`.
+
+The default value of this setting is `false`.
+
+{{< note >}}
+Within a checkpoint, only the span ID has a checkpoint, meaning spans do not survive a node restart. If the node restarts, the parent span information will be lost, and new spans will be generated for the flows. The root span after the node restart won't know who the parent span was before the node restart.
```

Comment:
Slight change to first sentence...

When checkpointing only the span id is checkpointed, meaning spans do not...

Reply:
Done